renderer.php

```diff
@@ -98,10 +98,22 @@ public function formulation_and_controls(question_attempt $qa, question_display_
                 [$iframeid, $qa->get_field_prefix() . constants::QT_VAR_RESPONSE]
             );
 
-            return <<<EOA
-    <input type="hidden" name="{$autosavehintname}" id="{$autosavehintid}" value="0">
-    <iframe id="{$iframeid}" srcdoc="{$iframesrc}"></iframe>
-EOA;
+            $result = '';
+            if (has_capability(constants::ROLE_VIEW_DETAILS, $options->context)) {
+                $detailsurl = new moodle_url(
+                    '/question/type/questionpy/attemptdetails.php',
+                    ['attemptid' => $qa->get_database_id()]
+                );
+                $result .= "<a class='qpy-details-link' href='{$detailsurl->out()}'>"
+                    . get_string('attempt_detail_link', 'qtype_questionpy') . '</a>';
+            }
```

Comment:
Ich fände es nutzerfreundlicher, wenn der Link
1. in einem neuen Tab geöffnet werden würde.
2. nicht angezeigt wird, sobald man bereits auf der Detail-Seite ist. (Das könnte über Moodles `me()` umgesetzt werden.)

Reply:
Für letzteres habe ich `question_display_options` erweitert, das schien mir etwas cleaner